src/pages/bylaws.js

```diff
@@ -389,7 +389,9 @@ const Bylaws = () => {
         <p>
           {' '}
           A vacancy in any office may be filled for the unexpired portion of the
-          term by a majority vote of the Board at any meeting.
+          term by a majority vote of the Board at any meeting. Each member of the
+          Board will cast one vote per vacancy. In case of tie, the President
+          will have a casting vote.
```

Comment:
deciding instead of casting?

Reply:
Had looked up the exact term: https://en.wikipedia.org/wiki/Casting_vote